Dockerfile.build

```diff
@@ -50,6 +50,20 @@ EXPOSE 80
 
 COPY --from=build /code/dataline-webapp/build /usr/share/nginx/html
 
+####################
+# Build seed image #
+####################
+FROM ubuntu:20.04 AS seed
```

Comment:
can you use `scratch` instead?

Reply:
talked offline. scratch doesn't work well with docker-compose. specifically docker-compose needs the container to have a `CMD` directive. scratch doesn't have one and there's no no-hacky way of adding one. went with `alpine` which is still a small image but has the default CMD we need so pay nicely with docker-compose.